include/Dialect/Pulse/Utils/Utils.h

```diff
@@ -32,11 +32,21 @@ namespace mlir {
 class Operation;
 } // namespace mlir
 
+using CallSequenceStack_t = std::deque<mlir::pulse::CallSequenceOp>;
+
 namespace mlir::pulse {
 
 Waveform_CreateOp getWaveformOp(PlayOp pulsePlayOp,
                                 CallSequenceOp callSequenceOp);
 
+Waveform_CreateOp getWaveformOp(PlayOp pulsePlayOp,
+                                CallSequenceStack_t &callSequenceOpStack);
```

Comment:
Can this be made const (both parameters and/or the method itself}?

Reply:
Yeah, and I just realized these are free methods, not part of a class. So trying to make them const doesn't make sense. So it's fine to ignore that. I'll mark this as resolved.